action.yml

```
@@ -94,23 +98,28 @@ runs:
       OSG_SERIES: ${{ inputs.osg_series }}
       REGISTRY: ${{ inputs.registry_url}}
       TIMESTAMP_TAG: ${{ inputs.timestamp_tag }}
+      TAG_OVERRIDE: ${{ inputs.image_tag }}
     shell: bash
     run: |
       docker_repo="$REGISTRY/${{ steps.generate-image-name.outputs.image }}"
       tag_list=()
-      if [[ -n $TIMESTAMP_TAG ]]; then
-        # grab the YYYYMMDD-hhmm timestamp
-        TIMESTAMP=${TIMESTAMP_TAG: -13}
+      if [[ -n "$TAG_OVERRIDE" ]]; then
+        tag_list+=("$docker_repo":"$TAG_OVERRIDE")
       else
-        TIMESTAMP=$(date +%Y%m%d-%H%M)
-      fi
-      for image_tag in "$OSG_SERIES-$REPO" "$OSG_SERIES-$REPO-$TIMESTAMP"; do
-        tag_list+=("$docker_repo":"$image_tag")
-        if [[ $OSG_SERIES == '3.5' ]]; then
-          tag_list+=("$docker_repo":"$REPO")
-          tag_list+=("$docker_repo":"$REPO-$TIMESTAMP")
+        if [[ -n $TIMESTAMP_TAG ]]; then
+          # grab the YYYYMMDD-hhmm timestamp
+          TIMESTAMP=${TIMESTAMP_TAG: -13}
```

```suggestion
          TIMESTAMP=${TIMESTAMP_TAG}
```
I do not know what the `: -13` is supposed to do... it appears to be a no-op when I try it

---

It appears to remove the first 13 characters from the string?
```bash
$ TIMESTAMP_TAG=24-el9-release-20250801
$ echo ${TIMESTAMP_TAG: -13}
ease-20250801
```
I do not know why this would be wanted though, it's probably fine to remove it

---

Ah, it seems to remove all _but_ the last 13 characters (i.e. the timestamp part) from the string, but does nothing if the string is not longer than 13 characters. The space is there to distinguish it from `${TIMESTAMP_TAG:-13}` which produces `13` if TIMESTAMP_TAG is undefined or empty.

So it actually does have a function (letting you propagate the timestamp from one step to the next). Sorry about the suggestion; can you add it back please?